packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -0,0 +1,465 @@
+import {Heading} from '@twilio-paste/heading';
+import {Radio, RadioGroup} from '@twilio-paste/radio-group';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {NewIcon} from '@twilio-paste/icons/esm/NewIcon';
+import {ProductVoiceIcon} from '@twilio-paste/icons/esm/ProductVoiceIcon';
+import {ProductElasticSIPTrunkingIcon} from '@twilio-paste/icons/esm/ProductElasticSIPTrunkingIcon';
+import {Grid, Column} from '@twilio-paste/grid';
+
+import {Truncate} from '@twilio-paste/truncate';
+import {styled, css} from '@twilio-paste/styling-library';
+
+import {Box} from '@twilio-paste/box';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Stack} from '@twilio-paste/stack';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {useUID} from '@twilio-paste/uid-library';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {CodeHighlight} from '../../../components/CodeHighlight';
+import {tableExample, betaFeatureExample, settingsAndProducts} from '../../../component-examples/BadgeExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/badge"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/badge/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Badge"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Badge"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/badge"
+  storybookUrl="/?path=/story/components-badge--all-badges"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Badge
+
+A badge is a small chunk of information that describes an object in a visually highlighted way. You can think of badges as a 1-3 word description of a more complex object that is highlighted to draw visual focus. Badges should be used to describe static attributes.
```

Comment:
We could probably add a quick snippet here about how badges are for static attributes (like "beta"), while the Status Pattern is used to indicate the the dynamic state of an object (like "in progress").